<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserCreateOrderWindow</class>
 <widget class="QDialog" name="UserCreateOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserCreateOrderWindow</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>寄件号码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>收件号码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>寄件地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>收件地址：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>241</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--------------------------------</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加货物</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>寄件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>货物列表清单（双击删除对应项）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货物id：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货物名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货物价格：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>货物数量：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
